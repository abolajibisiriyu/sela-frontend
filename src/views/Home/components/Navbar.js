import React from "react";
import NavBar from "../../../blocks/NavBar";

function HomeNavbar(props) {
  return (
    <NavBar>
      <NavBar.Logo href="#">SELA</NavBar.Logo>
    </NavBar>
  );
}

export default HomeNavbar;
