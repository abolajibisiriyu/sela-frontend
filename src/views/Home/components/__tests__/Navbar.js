import React from "react";
import renderer from "react-test-renderer";

import Navbar from "../Navbar";

describe("Navbar Component", () => {
  it("renders correctly", () => {
    const component = renderer.create(<Navbar />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
