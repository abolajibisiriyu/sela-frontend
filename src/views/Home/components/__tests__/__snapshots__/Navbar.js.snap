// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component renders correctly 1`] = `
<nav
  className="sc-bwzfXH cusHsC"
>
  <a
    className="sc-bdVaJa fJhYRb"
    href="#"
  >
    SELA
  </a>
</nav>
`;
